### PR TITLE
created models

### PR DESCRIPTION
**Pull request by Chrishnikawya**

### CODE REVIEW

**Comment by Chrishnikawya** [comment]
create the models

File: LibrarySystemBackend/LibrarySystem/LibrarySystem/Models/Author.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿namespace LibrarySystem.Models
+{
+    public class Author
+    {
+        public int AuthorID { get; set; }
+        public string AuthorName { get; set; } = string.Empty;
```
Comment (DiliniShanika92) [inline LibrarySystemBackend/LibrarySystem/LibrarySystem/Models/Author.cs]: No need to have  "= string.Empty"

File: LibrarySystemBackend/LibrarySystem/LibrarySystem/Models/Author.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+        public string AuthorAddress { get; set; } = string.Empty;
+        public string AuthorEmail { get; set; } = string.Empty;
+        public ICollection<Book> Books { get; set; }
```
Comment (DiliniShanika92) [inline LibrarySystemBackend/LibrarySystem/LibrarySystem/Models/Author.cs]: public  virtual ICollection<Book> Books { get; set; }

File: LibrarySystemBackend/LibrarySystem/LibrarySystem/Models/Book.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
+﻿namespace LibrarySystem.Models
+{
+    public class Book
+    {
+        public int BookID { get; set; }
+        public string BookName { get; set; } = string.Empty;
+        public string Category { get; set; } = string.Empty;
+        public int AuthorID { get; set; }
+        public int PublisherID { get; set; }
```
Comment (DiliniShanika92) [inline LibrarySystemBackend/LibrarySystem/LibrarySystem/Models/Book.cs:11]: public int AuthorID { get; set; }
public virtual Author Author { get; set; }
public int PublisherID { get; set; }
public virtual Publisher Publisher { get; set; }

File: LibrarySystemBackend/LibrarySystem/LibrarySystem/Models/Member.cs [hunk 4]
```diff
@@ -0,0 +1,11 @@
+﻿namespace LibrarySystem.Models
+{
+    public class Member
+    {
+        public int MemberID { get; set; }
+        public string MemberName { get; set; } = string.Empty;
+        public string Email { get; set; } = string.Empty;
```
Comment (DiliniShanika92) [inline LibrarySystemBackend/LibrarySystem/LibrarySystem/Models/Member.cs]: Rename to MemberEmail and MemberPhone number.

Comment (DiliniShanika92) [reply LibrarySystemBackend/LibrarySystem/LibrarySystem/Models/Member.cs]: By having unique names in every model we can easily identify them

File: LibrarySystemBackend/LibrarySystem/LibrarySystem/Models/Staff.cs [hunk 5]
```diff
@@ -0,0 +1,11 @@
+﻿namespace LibrarySystem.Models
+{
+    public class Staff
+    {
+        public int StaffID { get; set; }
+        public string StaffName { get; set; } = string.Empty;
+        public string Email { get; set;} = string.Empty;
```
Comment (DiliniShanika92) [inline LibrarySystemBackend/LibrarySystem/LibrarySystem/Models/Staff.cs]: Rename to StaffEmail

**Review by DiliniShanika92 — CHANGES_REQUESTED** [review]
Please done the changes mentioned in comments.

File: LibrarySystemBackend/LibrarySystem/LibrarySystem/Models/Author.cs (hunk 2 above)
Comment (Chrishnikawya) [reply LibrarySystemBackend/LibrarySystem/LibrarySystem/Models/Author.cs]: Done

File: LibrarySystemBackend/LibrarySystem/LibrarySystem/Models/Staff.cs (hunk 5 above)
Comment (Chrishnikawya) [reply LibrarySystemBackend/LibrarySystem/LibrarySystem/Models/Staff.cs]: Done

**Pull request merged**